dotnetv3/Organizations/README.md

```diff
@@ -0,0 +1,55 @@
+# AWS Organizations code examples for .NET
```

Comment:
# Organizations code examples for the SDK for .NET

Reply:
Fixed.